src/ui/atoms/StatusLabel/StatusLabel.scss

```diff
@@ -0,0 +1,48 @@
+.status-label {
+  display: inline-flex;
+  padding: 2px 12px;
+  font-size: 14px;
+  font-weight: bold;
+  border-radius: 16px;
+  white-space: nowrap;
+  text-transform: capitalize;
+  cursor: default;
+  border-width: 1px;
+  border-style: solid;
+
+  &-primary {
+    background-color: #e7f1ff;
```

Comment:
These color codes should be put inside a new file named colors.sass as variables:

$primary: #e7f1ff;
$secondary: #f4f4f4;
...

And import the colors.sass file here 

@import '@sass/colors.sass'